docs/moderator-handbook.md

```diff
@@ -147,6 +147,12 @@ There are also situations where the contributor is trying to add a link to their
 
 You can close these invalid pull requests and reply to them with these [reply templates](reply-templates.md#closing-invalid-pull-requests).
 
+#### Pull Request Pro Tip:
```

Comment:
This is a bit ambiguous, maybe name the section more clearly as related to filtering pull requests.
```suggestion
#### Filtering Pull Requests
```